transifex/native/django/management/utils/push.py

```diff
@@ -167,6 +172,27 @@ def push_strings(self):
         status_code, response_content = tx.push_source_strings(
             self.string_collection.strings.values(), self.purge
         )
+
+        if self.no_wait:
+            Color.echo('Queued')
+            return
+
+        job_url = response_content.get('data').get('links').get('job')
+        status = 'pending'
+        while status in ['pending', 'processing']:
+            time.sleep(1)
+            status_code, response_content = tx.get_push_status(job_url)
+            new_status = response_content.get('data').get('status')
```

Comment:
same

Reply:
If status is `None` then the while loop will never end...